README.md

```diff
@@ -6,46 +6,24 @@
 
 As of August 14, 2020, the [EPUB 3 Working Group](https://www.w3.org/publishing/groups/epub-wg) is now the maintainer of the EPUB 3 specifications.
 
-## EPUB 3.3
+## Current version of EPUB: EPUB 3.3
 
-Editor's Drafts of documents under active development:
+On the 25th of May, 2023, the [EPUB 3 Working Group](https://www.w3.org/publishing/groups/epub-wg) published three 
+EPUB 3.3 Recommendations ([EPUB 3.3](https://www.w3.org/TR/epub-33/), [EPUB Reading Systems 3.3](https://www.w3.org/TR/epub-rs-33/), and [EPUB Accessibility 1.1](https://www.w3.org/TR/epub-a11y-11/)), as well as a number of Working Group Notes. For a complete list of the publications by the Working Group, see the
+[separate index page](https://www.w3.org/publishing/epub33/).
```

Comment:
Sorry, I'm going to nitpick one other thing here. Technically we didn't publish the notes on the 25th, but that's what the sentence suggests. Could we just break out the RECs into a list (easier to read/click) and maybe just say there are notes in the pointer to the index?

```suggestion
On the 25th of May, 2023, the [EPUB 3 Working Group](https://www.w3.org/publishing/groups/epub-wg) published the following EPUB 3.3 Recommendations:

- [EPUB 3.3](https://www.w3.org/TR/epub-33/),
- [EPUB Reading Systems 3.3](https://www.w3.org/TR/epub-rs-33/), and
- [EPUB Accessibility 1.1](https://www.w3.org/TR/epub-a11y-11/)).

For a complete list of publications by the Working Group, including all Working Group Notes, see the
[separate index page](https://www.w3.org/publishing/epub33/).
```